pandas/core/base.py

```diff
@@ -957,10 +957,16 @@ def _ndarray_values(self):
     def empty(self):
         return not self.size
 
-    def max(self):
+    def max(self, axis=None, skipna=True):
```

Comment:
use **kwargs I think is consistent with how we do this elsewhere, IOW we don't explicity list the numpy support args (basically remove axis)

Reply:
Adding kwargs kinda makes sense (as long as we dont ignore them silently), but taking axis out seems weird; a big part of the goal is to make the signatures match across the different classes

Reply:
right, the series reduction method have `axis` in their sigatures

```
>>> inspect.getargspec(pd.Series.max)
ArgSpec(args=['self', 'axis', 'skipna', 'level', 'numeric_only'], varargs=None, keywords='kwargs', defaults=(None, None, None, None))
```